src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -6,29 +6,35 @@
 
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 
 public class DrivetrainSubsystem extends SubsystemBase {
   /** Creates a new DrivetrainSubsystem. */
 
-  public WPI_TalonFX leftTop;
-  public WPI_TalonFX leftBottom;
-  public WPI_TalonFX rightTop;
-  public WPI_TalonFX rightBottom;
+  public WPI_TalonFX leftTopMotor;
+  public WPI_TalonFX leftBottomMotor;
+  public WPI_TalonFX rightTopMotor;
+  public WPI_TalonFX rightBottomMotor;
 
   public DrivetrainSubsystem() {
 
-    leftTop = new WPI_TalonFX(Constants.DrivetrainConstants.LEFT_TOP_MOTOR);
-    leftBottom = new WPI_TalonFX(Constants.DrivetrainConstants.LEFT_BOTTOM_MOTOR);
-    rightTop = new WPI_TalonFX(Constants.DrivetrainConstants.RIGHT_TOP_MOTOR);
-    rightBottom = new WPI_TalonFX(Constants.DrivetrainConstants.RIGHT_BOTTOM_MOTOR);
-
+    leftTopMotor = new WPI_TalonFX(Constants.DrivetrainConstants.LEFT_TOP_MOTOR);
+    leftBottomMotor = new WPI_TalonFX(Constants.DrivetrainConstants.LEFT_BOTTOM_MOTOR);
+    rightTopMotor = new WPI_TalonFX(Constants.DrivetrainConstants.RIGHT_TOP_MOTOR);
+    rightBottomMotor = new WPI_TalonFX(Constants.DrivetrainConstants.RIGHT_BOTTOM_MOTOR);
 
   }
 
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+    double leftTopMotorPos = leftTopMotor.getSelectedSensorPosition();
+    double leftBottomMotorPos = leftBottomMotor.getSelectedSensorPosition();
+    double rightTopMotorPos = rightTopMotor.getSelectedSensorPosition();
+    double rightBottomMotorPos = rightTopMotor.getSelectedSensorPosition();
+
+    SmartDashboard.putNumber("leftTopEncoder", leftTopMotorPos);
```

Comment:
Why just the left top motor?  This would be a great place to compute the average, store it in an instance variable, and put it on the SmartDashboard.